<ui version="4.0" >
 <class>MsgForm</class>
 <widget class="QWidget" name="MsgForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Messages:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget" >
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Source string:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="sourceEdit" />
         </item>
        </layout>
       </item>
       <item row="0" column="1" >
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Source annotation:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="sourceAnnotEdit" />
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Translation:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="translEdit" />
         </item>
        </layout>
       </item>
       <item row="1" column="1" >
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Translator's annotation:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="translAnnotEdit" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
